ui/static/js/app.js

```diff
@@ -167,11 +167,24 @@ function markEntryAsRead(element) {
     }
 }
 
+// Send the Ajax request to refresh all feeds in the background
+function handleRefreshAllFeeds() {
+    let url = "/feeds/refresh";
```

Comment:
Hardcoding the URL should be avoided because some people are running Miniflux within a subfolder. You could do the same thing as the function below:

```
let url = document.body.dataset.refreshAllFeedsUrl;
```

And let the router generate the URL by updating the body element in layout.html.